backend/app/cmd/server.go

```diff
@@ -366,7 +369,8 @@ func (s *ServerCommand) newServerApp() (*serverApp, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to make avatar store")
 	}
-	authenticator, err := s.makeAuthenticator(dataService, avatarStore, adminStore)
+	refreshCache := newAuthRefreshCache()
```

Comment:
I'd suggest to name it authRefreshCache because latter one it is hard to figure what refreshCache was about